field_records/ubna_2023.csv

```diff
@@ -150,7 +150,14 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
 2023-11-27T15:39:00 , 2023-12-04T16:04:00 , M , 001 , Central Pond , 47° 39' 16.860'' N , 122° 17' 34.140'' W , 192000 , Medium , None , None , 4.126 , 3.605 , VK , MS , MS , recover-20231204 , Ikea Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
-2023-12-04T15:13:00 , (DATE-TIME) , C , 002 , Foliage , 47° 39' 18.408'' N , 122° 17' 26.832'' W , 192000 , Medium , None , None , 4.201 , (VOLTAGE) , VK , AK , MS , (RECOVER-DATE) , Ikea Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
-2023-12-04T15:12:00 , (DATE-TIME) , D , 007 , Telephone Field , 47° 39' 25.212'' N , 122° 17' 29.802'' W , 192000 , Medium , None , None , 4.359 , (VOLTAGE) , VK , AK , MS , (RECOVER-DATE) , Ikea Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
-2023-12-04T15:12:00 , (DATE-TIME) , K , 010 , E18 Bridge , 47° 39' 21.582'' N , 122° 17' 49.020'' W , 192000 , Medium , None , None , 4.133 , (VOLTAGE) , VK , AK , MS , (RECOVER-DATE) , Panasonic Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
-2023-12-04T15:13:00 , (DATE-TIME) , I , 012 , Central Pond , 47° 39' 16.788'' N , 122° 17' 34.470'' W , 192000 , Medium , None , None , 4.126 , (VOLTAGE) , VK , AK , MS , (RECOVER-DATE) , Ikea Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
+2023-12-04T15:13:00 , 2023-12-13T18:07:00 , C , 002 , Foliage , 47° 39' 18.408'' N , 122° 17' 26.832'' W , 192000 , Medium , None , None , 4.201 , (VOLTAGE) , VK , AK , MS , recover-20231213 , Ikea Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF; bag wet; recorder wet; Ending voltage not recorded;
+2023-12-04T15:12:00 , 2023-12-13T18:07:00 , D , 007 , Telephone Field , 47° 39' 25.212'' N , 122° 17' 29.802'' W , 192000 , Medium , None , None , 4.359 , 3.569 , VK , AK , MS , recover-20231213 , Ikea Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
+2023-12-04T15:12:00 , 2023-12-13T17:01:00 , K , 010 , E18 Bridge , 47° 39' 21.582'' N , 122° 17' 49.020'' W , 192000 , Medium , None , None , 4.133 , 3.702 , VK , AK , MS , recover-20231213 , Panasonic Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
+2023-12-04T15:13:00 , 2023-12-13T17:30:00 , I , 012 , Central Pond , 47° 39' 16.788'' N , 122° 17' 34.470'' W , 192000 , Medium , None , None , 4.126 , 3.442 , VK , AK , MS , recover-20231213 , Ikea Batteries 0:00-24:00 UTC; Configured to 5m ON/OFF;
+2023-12-13T16:05:00 , (DATE-TIME) , M , 009 , (SITE-NAME) , (LATITUDE) , (LONGITUDE) , 192000 , Medium , None , None , 4.157 , (VOLTAGE) , AK , MS , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 0:00-16:00 UTC; Not deployed; Wire issue; Audiomoth M’s wires were burnt and the bag melted;
+2023-12-13T16:06:00 , (DATE-TIME) , A , 018 , Telephone Field , 47° 39' 25.218'' N , 122° 17' 29.520'' W , 192000 , Medium , None , None , 4.360 , (VOLTAGE) , VK , MS , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 0:00-16:00 UTC; UBNA_001 was put inside TEL Audiomoth A, but due to errors we replaced it with UBNA_018; UBNA_001 has recordings from lab to field; nothing relevant delete these recordings when able; 
```

Comment:
```suggestion
2023-12-13T16:06:00 , (DATE-TIME) , A , 018 , Telephone Field , 47° 39' 25.218'' N , 122° 17' 29.520'' W , 192000 , Medium , None , None , 4.360 , (VOLTAGE) , VK , MS , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 0:00-16:00 UTC; UBNA_001 was put inside TEL Audiomoth A, but due to errors we replaced it with UBNA_018; UBNA_001 had recordings from lab to field that were deleted because they were not relevant; 
```

The files were deleted so you can change the notes to this.